test/compile/scale.test.ts

```diff
@@ -389,11 +389,51 @@ describe('Scale', function() {
   });
 
   describe('rangeMixins()', function() {
-    // FIXME
+    describe('row', function() {
+      // TODO:
+    });
+
+    describe('column', function() {
+      // TODO:
+    });
+
+    describe('x', function() {
+      // TODO: X
+    });
+
+    describe('y', function() {
```

Comment:
That's irrelevant for `rangeMixins()`. 

`rangeMixins()` of `x` and `y` scales are **independent**.